composer.json

```diff
@@ -118,6 +118,7 @@
     "conflict": {
         "doctrine/dbal": "2.7.0",
         "doctrine/lexer": "1.0.0",
+        "symfony/doctrine-bridge": "< 3.4.0",
```

Comment:
The symfony/doctrine-bridge is no requirement of sulu but it conflicts with symfony/form 3.4 when installed on prefer lowest.